package.json

```diff
@@ -18,16 +18,84 @@
     "scripts": {
         "clean": "lerna clean",
         "test": "lerna run test",
+        "appwebtest": "sh ./app_web_test.sh",
+        "appapitest": "sh ./app_api_test.sh",
         "lint": "lerna run lint --parallel",
         "format": "yarn prettier --write",
         "prettier": "prettier --ignore-path .gitignore \"**/*.+(ts|tsx|json)\"",
         "cypress:open": "cypress open --env AUTH_MODE=LOCAL",
-        "cypress:run": "cypress run --env AUTH_MODE=LOCAL"
+        "cypress:run": "rm -rf .nyc_output && ./node_modules/.bin/cypress run --env AUTH_MODE=LOCAL --headless --browser chrome",
+        "test:coverage": "yarn run appwebtest && yarn run appapitest && yarn run cypress:run",
+        "pretest:coverage": "rm -rf .nyc_output",
+        "posttest:coverage": "yarn run report:combined",
+        "mkdir:reports": "mkdir -p reports",
+        "precopy:reports": "yarn run mkdir:reports",
+        "copy:reports": "cp ./coverage-cypress/coverage-final.json ./reports/from-cypress.json && cp ./services/app-web/coverage/coverage-final.json ./reports/from-jest.json && cp ./services/app-api/coverage/coverage-final.json ./reports/from-api.json",
+        "precombine:reports": "yarn run copy:reports && mkdir -p .nyc_output",
+        "combine:reports": "npx nyc merge reports && mv coverage.json .nyc_output/out.json",
+        "prereport:combined": "yarn run combine:reports",
+        "report:combined": "mkdir -p coverage-all && npx nyc report --reporter lcov --reporter text --reporter json-summary --report-dir coverage-all"
     },
     "repository": {
         "type": "git",
         "url": "git+https://github.com/CMSgov/managed-care-review.git"
     },
+    "nyc": {
+        "report-dir": "coverage-cypress",
+        "exclude": [
+            "/*.stories.tsx",
+            "**/src/s3",
+            "**/src/api",
+            "**/src/testHelpers",
+            "**/src/localAuth",
+            "**/src/gen",
+            "**/src/components/Header/StateIcon/StateIcon.tsx",
```

Comment:
I think we could add `/constants` folder, `TraceContext.tsx`, `useTealium.ts` for app-web. And I think we could exclude `awsParameterStore.ts` for app-api. 

I see what you are saying about the low level auth functions maybe being an option to exclude, though at the same time it feels like we could be testing some of those using mocks so I'm of two minds about. We could at least be testing our error handling. I'm thinking lets leave it for now

Reply:
added